[docs] Rename Expo client to Expo Go (#11747)
expo/expo: docs/pages/introduction/walkthrough.md

Upon publishing you are given a persistent URL that you can share with colleagues, in this case it was [https://expo.io/@notbrent/blearexp](https://expo.io/@notbrent/blearexp). This is determined by your Expo account username and the `slug` field in your project `app.json`.

> On iOS, only you can open projects that you have built unless you have a [priority plan](https://expo.io/developer-services), in which case your teammates can open your projects as well. Another option to open any published managed app from within the Expo Go app is to do a custom build of the Expo iOS client. [Read more about that here](../guides/adhoc-builds.md).

Reviewer: ```suggestion
On iOS, only you can open projects that you have built unless you have a [priority plan](https://expo.io/developer-services), in which case your teammates can open your projects as well. Another option to open any published managed app is to do a custom build of an Expo iOS client. [Read more about that here](../guides/adhoc-builds.md).
```
